desktop/core/src/desktop/cm_environment.py

```diff
@@ -0,0 +1,225 @@
+# Licensed to Cloudera, Inc. under one
```

Comment:
This file seems duplicated as https://github.com/cloudera/hue/blob/master/tools/ops/script_runner/lib/cm_environment.py. Might be just need update import statement to the existing files.

Reply:
Wanted this to be in the Hue's default location.  Will be removing these unneeded files, once all of the informations are moved which are used.

There are still some more scripts which we use, going over them and then will sort these out.